release notes/v0.57.0.md

```diff
@@ -0,0 +1,180 @@
+
+    // Expecting this to fail as we have typed 'testlogin' into the input instead of 'foo'
+    await expect(page.locator('input[name="login"]')).toHaveValue("foo");
+  } finally {
+    await page.close();
+  }
+}
+```
+
+It is currently available as part of the [jslib.k6.io](https://jslib.k6.io) repository and is available to use in your k6 tests by adding the following import:
+
+```javascript
+import { expect } from "https://jslib.k6.io/k6-testing/0.2.0/index.js";
+```
+
+Try it out and give us feedback or contribute to the project on the [k6-jslib-testing repository](https://github.com/grafana/k6-jslib-testing)!
+
+
+## UX improvements and enhancements
+
+### New `asObjects` option in `k6/experimental/csv` module [#4295](https://github.com/grafana/k6/pull/4295)
+
+The CSV module's parsing operations now support the `asObjects` option, which enables parsing CSV data into JavaScript objects instead of arrays of strings (the default behavior).
+
+When `asObjects` is set to `true`, the module parses CSV data into objects where:
+- Column names from the header row become object keys
+- Column values become the corresponding object values
+- An error is thrown if no header row exists or if options modify the parsing start point
+
+With the option set to `true`, 
+
+```js
+import http from 'k6/http';
+import csv from 'k6/experimental/csv';
+
+const csvData = csv.parse('data.csv', { asObjects: true });
+```
+
+the following CSV file:
+
+```csv
+name,age,city
+John,30,New York
+Jane,25,Los Angeles
+```
+
+will be parsed into the following JavaScript objects:
+
+```js
+[
+  { name: 'John', age: '30', city: 'New York' },
+  { name: 'Jane', age: '25', city: 'Los Angeles' },
+]
+```
+
+See the [csv module's documentation](https://grafana.com/docs/k6/latest/javascript-api/k6-experimental/csv/) for more information.
+
+- [#4176](https://github.com/grafana/k6/pull/4176) Warns on using shorthand options when that override scenarios.
+- [#4293](https://github.com/grafana/k6/pull/4293) Renames browser data directory name prefix from `xk6-browser-data-` to `k6browser-data-`.
+- [#4513](https://github.com/grafana/k6/pull/4513) Adds support for `file` scheme URLs across file loading APIs - `open`, `k6/experimental/fs.open` and `k6/net/grpc.Client#load`.
+- [#4517](https://github.com/grafana/k6/pull/4517) Switches from the legacy examples to quickpizza.grafana.com.
+
+_Format as `<number> <present_verb> <object>. <credit>`_:
+
+- _`#999` Gives terminal output prettier printing. Thanks to `@person` for the help!_
+- `#pr` `<description>`
+- `#pr` `<description>`
+
+## Bug fixes
+
+- [#4174](https://github.com/grafana/k6/pull/4174) Fixes an NPD during a click, which could occur when either the load generator or chrome instance is under a lot of load.
+- [#4184](https://github.com/grafana/k6/pull/4184) Fixes some browser Windows tests.
+- [#4192](https://github.com/grafana/k6/pull/4192) Fixes a potential memory leak.
```

Comment:
```suggestion
- [#4192](https://github.com/grafana/k6/pull/4192) Fixes a memory leak when .... on component XYZ.
```
It would be good to give a better insight of where the problem could _potentially_ be.

Reply:
```suggestion
- [#4192](https://github.com/grafana/k6/pull/4192) Fixes a memory leak in general event handling between components.
```